Source/Events.Store.MongoDB/Processing/Streams/StreamProcessorStateDiscriminatorConvention.cs

```diff
@@ -33,7 +33,7 @@ class StreamProcessorStateDiscriminatorConvention : IDiscriminatorConvention
 
         /// <summary>
         /// Gets the correct type when deserializing objects from  <see cref="AbstractStreamProcessorState"/> collection
-        /// depending on the given nominal type or "Partitioned" field.
+        /// depending on the "Partitioned" field. Also saves the _id field for the exception if needed.
```

Comment:
What does "Also saves the _id field for the exception if needed." mean?

If you are reffering to the `ObjectId` that is stored in the loop to throw meaningful exceptions, this should not be described in the documentation summary, as it is purely internal to the method.

Reply:
Fixd, made it just `inheritdoc`